Tests/SwiftDocCTests/Infrastructure/PathHierarchyTests.swift

```diff
@@ -1239,7 +1239,25 @@ class PathHierarchyTests: XCTestCase {
         XCTAssertEqual(paths["s:8ShapeKit14OverloadedEnumO19firstTestMemberNameySdSaySdGF"],
                        "/ShapeKit/OverloadedEnum/firstTestMemberName(_:)-88rbf")
     }
-    
+
+    func testOverloadGroupSymbolsResolveLinksWithoutHash() throws {
```

Comment:
Please also add a new test in `ExternalPathHierarchyResolverTests`. For all new link behavior we want to verify that the a link that resolves locally also resolves externally.